debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -1103,6 +1103,32 @@ public void shouldReceiveChangesForTableWithoutPrimaryKey() throws Exception{
         assertRecordSchemaAndValues(expectedAfter, deletedRecord, Envelope.FieldName.AFTER);
     }
 
+    @Test()
+    public void testPassingStreamParams() throws Exception {
+        // Verify that passing stream parameters works by using the WAL2JSON add-tables parameter which acts as a
+        // whitelist.
+
+        PostgresConnectorConfig config = new PostgresConnectorConfig(TestHelper.defaultConfig()
+                .with(PostgresConnectorConfig.PLUGIN_NAME, "wal2json")
+                .with(PostgresConnectorConfig.STREAM_PARAMS, "add-tables=s1.should_stream")
```

Comment:
Could you add another test for making sure that passing multiple parameters works, too?

Reply:
Good suggestion. I added a new test to cover multiple parameters and parameters using multiple values.

Reply:
Thanks!